src/herbivoresAndCarnivores.js

```diff
@@ -1,16 +1,42 @@
 'use strict';
-
 class Animal {
-  // write your code here
-}
+  constructor(name) {
+    this.name = name;
+    this.health = 100;
+    Animal.alive.push(this);
+  };
+};
+
+Animal.alive = [];
 
 class Herbivore extends Animal {
-  // write your code here
-}
+  constructor(name) {
+    super(name);
+    this.hidden = false;
+  }
+
+  hide() {
+    if (this.hidden === false) {
+      this.hidden = true;
+    } else if (this.hidden === true) {
+      this.hidden = false;
+    };
```

Comment:
```suggestion

      this.hidden = !this.hidden;

```
цей код робить аналогічне